docs/formats.md

```
@@ -0,0 +1,101 @@
+    - **BED4**: A BED file where each feature is described by chrom, start, end, and name. (e.g. `chr1  11873  14409  VLADIMIR`)
+    - **BED6**: A BED file where each feature is described by chrom, start, end, name, score, and strand. (e.g. `chr1 11873 14409 VLADIMIR 0 +`)
+    - **BED12**: A BED file where each feature is described by all twelve columns listed above. (Default option in all tools)
+    - **BED12+11**: A complete BED file including **required** fields and optionals.
+
+### Color
+
+**Uniqueness** Colors:
+
+| Colour | Description |
+|---|---|
+| <img src="images/uniquetranscript.svg" width="25"> | Peptide is unique to single gene AND single transcript |
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)